### PR TITLE
Fixes#406 Position of version link is now fixed to bottom

### LINKED ISSUE #406: Fix Version link to bottom
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**
image

**Expected behavior:**
It should be at bottom

I am working on this.

### PR DESCRIPTION
**Pull request by ParthS007**
Fixes #406 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] I have added necessary documentation (if appropriate)

#### Changes proposed in this pull request:

-  Version Link is fixed to the bottom.
- [https://gentle-springs-78873.herokuapp.com/](https://gentle-springs-78873.herokuapp.com/)
image

### CODE REVIEW

**Comment by harshit98** [comment]
@ParthS007 Provide demo link with the PR.

**Comment by ParthS007** [comment]
@harshit98 Please check I have updated the link 👍

File: app/templates/layout.html [hunk 1]
```diff
@@ -20,9 +20,9 @@
     <script src="{{ url_for('static', filename='js/script.js') }}"></script>
     <script src="{{ url_for('static', filename='js/form.js') }}"></script>
 
-    <div class="footer">
-        <strong>Version:</strong>
-        <a class="version"></a>
+    <div id="footer">
+        <span> <strong>Version: </strong></span>
+        <span> </span><a class="version"></a></span>
```
Comment (harshit98) [inline app/templates/layout.html]: Two times `</span>` is occurring.

**Comment by ParthS007** [comment]
@harshit98 Please check done the changes and updates deployment link.

**Pull request merged**